src/framework/preferences/messages/textFont_fr.json

```diff
@@ -1,5 +1,6 @@
 {
-    "textFont-default": "Défaut",
+    "textFont-default": "No Preference",
+    "textFont-system": "Device Font",
```

Comment:
@BlueSlug could you help with translating these two strings to French?

Reply:
@BlueSlug has recommended

-     "No Preference" => "Pas de préférence"
-     "Device Font" => "Police de l’appareil"